error-prone-contrib/src/test/java/tech/picnic/errorprone/bugpatterns/ImplicitBlockingFluxOperationTest.java

```diff
@@ -0,0 +1,147 @@
+package tech.picnic.errorprone.bugpatterns;
+
+import static com.google.common.base.Predicates.containsPattern;
+import static com.google.common.base.Predicates.not;
+
+import com.google.errorprone.BugCheckerRefactoringTestHelper;
+import com.google.errorprone.BugCheckerRefactoringTestHelper.FixChoosers;
+import com.google.errorprone.BugCheckerRefactoringTestHelper.TestMode;
+import com.google.errorprone.CompilationTestHelper;
+import org.junit.jupiter.api.Test;
+import org.reactivestreams.Publisher;
+import reactor.core.CorePublisher;
+import reactor.core.publisher.Flux;
+
+final class ImplicitBlockingFluxOperationTest {
+  @Test
+  void identification() {
+    CompilationTestHelper.newInstance(ImplicitBlockingFluxOperation.class, getClass())
+        .addSourceLines(
+            "A.java",
+            "import reactor.core.publisher.Flux;",
+            "import tech.picnic.errorprone.bugpatterns.util.NotFlux;",
+            "",
+            "class A {",
+            "  void m() {",
+            "    // BUG: Diagnostic contains:",
+            "    Flux.just(1).toIterable();",
+            "    // BUG: Diagnostic contains:",
+            "    Flux.just(2).toStream();",
+            "    // BUG: Diagnostic contains:",
+            "    long count = Flux.just(3).toStream().count();",
+            "",
+            "    Flux.just(3).toStream(16);",
+            "    new NotFlux().toIterable();",
```

Comment:
You put quite some effort in this solution, nice! Luckily we can make it a bit easier for ourselves and insert the class as subclass. Something else we could do is add another class on the same instance of the `CompilationTestHelper`. For an example see the `JUnitMethodDeclarationTest#identification` where we call `addSourceLines` twice, once for a `A.java` and once with `B.java`. 

I pushed a suggestion based on your changes 🚀 !